rhel8/nvidia-driver

```diff
@@ -577,6 +575,27 @@ _start_vgpu_topology_daemon() {
     nvidia-topologyd
 }
 
+_resolve_kernel_type() {
+  if [ "${KERNEL_MODULE_TYPE}" == "proprietary" ]; then
+    KERNEL_TYPE=kernel
+    return 0
```

Comment:
nit: we could omit all of the `return 0` statements as reaching the end of this function will return 0 anyways.

Reply:
agreed